subsys/net/lib/openthread/platform/radio.c

```diff
@@ -1210,3 +1228,8 @@ otError otPlatRadioConfigureEnhAckProbing(otInstance *aInstance, otLinkMetrics a
 }
 
 #endif /* CONFIG_OPENTHREAD_LINK_METRICS_SUBJECT */
+
+uint16_t otPlatTimeGetXtalAccuracy()
+{
+	return CONFIG_CLOCK_CONTROL_NRF_ACCURACY;
```

Comment:
We can't use this define here, it's Nordic specific.

Reply:
I cannot find any universal way to get XtalAccuracy. Should I set it here at INT_MAX (which won't allow device to become a router if OT Time Sync is active, so it being default might be problematic), and use CONFIG_CLOCK_CONTROL_NRF_ACCURACY in some Nordic specific place?

Reply:
What about to create a new KConfig for OpenThread platform, and set the accuracy to `CONFIG_CLOCK_CONTROL_NRF_ACCURACY` when nRF5x device is used, else set it to 0?